app/src/main/res/layout/fragment_tabhome.xml

```diff
@@ -1,18 +1,191 @@
+                />
+            <Button
+                android:id="@+id/moreNowPlayingButton"
+                android:layout_width="@dimen/dp_60"
+                android:layout_height="@dimen/dp_30"
+                android:background="@color/colorOrange"
+                android:text="@string/more"
+                android:textColor="@color/colorWhite"
+                android:textStyle="bold"
+                android:layout_alignParentRight="true"
+                />
+        </RelativeLayout>
+        <android.support.v7.widget.RecyclerView
+            android:id="@+id/nowPlayingRecyclerView"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_below="@+id/containerNowPlaying"
+            />
+
+        <RelativeLayout
+            android:id="@+id/containerUpComing"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_below="@+id/nowPlayingRecyclerView"
+            android:gravity="center_vertical"
+            android:paddingLeft="@dimen/dp_15"
+            android:paddingRight="@dimen/dp_15"
+            android:layout_marginTop="10dp"
+            >
+            <TextView
+                android:id="@+id/upComingTextView"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:text="@string/upcoming"
+                android:textSize="@dimen/sp_20"
+                android:textStyle="bold"
+                android:textColor="@color/colorWhite"
+                android:layout_centerVertical="true"
+                android:layout_alignParentLeft="true"
+                />
+            <Button
+                android:id="@+id/moreUpComingButton"
+                android:layout_width="@dimen/dp_60"
+                android:layout_height="@dimen/dp_30"
+                android:background="@color/colorOrange"
+                android:text="@string/more"
+                android:textColor="@color/colorWhite"
+                android:textStyle="bold"
+                android:layout_alignParentRight="true"
+                />
+        </RelativeLayout>
+        <android.support.v7.widget.RecyclerView
+            android:id="@+id/upComingRecyclerView"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_below="@+id/containerUpComing"
+            />
+
+        <RelativeLayout
+            android:id="@+id/containerTopRated"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_below="@+id/upComingRecyclerView"
+            android:gravity="center_vertical"
+            android:paddingLeft="@dimen/dp_15"
+            android:paddingRight="@dimen/dp_15"
+            android:layout_marginTop="10dp"
+            >
+            <TextView
```

Comment:
WARNING: [RelativeOverlap] `@id/topRatedTextView` can overlap `@id/moreTopRatedButton` if @string/top_rated grows due to localized text expansion
 If relative layout has text or button items aligned to left and right sides they can overlap each other due to localized text expansion unless they have mutual constraints like `toEndOf`/`toStartOf`.